tools/preflight/preflight.go

```diff
@@ -446,6 +453,8 @@ func (o *Options) checkVolumeSnapshot(ctx context.Context) error {
 		waitOptions *wait.PodWaitOptions
 		err         error
 	)
+	updateVolSnapPodResourceReqs(o.PodResourceRequirements)
+	logVolumeSnapshotResReqs(o.Logger)
```

Comment:
instead such log functions.. print whole PreflightOptions struct in json format before running preflight check.
do the same for cleanup command